src/renderer/src/design/ui/Button/Button.tsx

```
@@ -0,0 +1,52 @@
+type ButtonProperty = 'DEFAULT' | 'DISABLED'
+
+type ButtonSize = 'SMALL'
+
+type Props = {
+  children: ReactNode
+  property: ButtonProperty
+  size?: ButtonSize
+  width?: CSSProperties['width']
+} & ButtonHTMLAttributes<HTMLButtonElement>
+
+const Button = ({ onClick, children, property, width, style, disabled }: Props) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)